examples/plot_score_alg.py

```diff
@@ -0,0 +1,210 @@
+# the Context class houses both data, apriori assumptions and other relevant data
+# that may be used in downstream structure learning algorithms.
+context = make_context().variables(data=data).build()
+
+# Alternatively, one could say specify some fixed edges.
+# Note that when specifying fixed edges, the resulting graph that is
+# learned is not necessarily a "pure CPDAG". In that, there are more
+# constraints than just the conditional independences. Therefore, one
+# should use caution when specifying fixed edges if they are interested
+# in leveraging ID or estimation algorithms that assume the learned
+# structure is a "pure CPDAG".
+
+# .. code-block::Python
+#   included_edges = nx.Graph([('x', 'y')])
+#   context = make_context().edges(include=included_edges).build()
+
+# %%
+# Run structure learning algorithm
+# --------------------------------
+# Now we are ready to run the SCORE algorithm. The method performs inference
+# in two phases. First it estimates the topological order of the nodes in the
+# graphs. This is done iteratively according to the following procedure:
+#
+# 1. SCORE estimates the Hessian of the logarithm of :math:`p(x)`,
+# with :math:`p(x)` the joint distribution of the nodes in the graph.
+#
+# 2. Let `H := Hessian(log p(x))`. SCORE selects a leaf in the graph by finding
+# the diagonal term of H with minimum variance,
+# i.e. by computing `np.argmin(np.var(np.diag(H))`.
+#
+# 3. SCORE removes the leaf in from the graph, and repeats steps from 1. to 3.
+# iteratively up to the source nodes.
+# Given the inferred topological order, SCORE prunes the graph by with all
+# edges admitted by such ordering, by doing sparse regression to choose the
+# relevant variables. Variable selection is done by thresholding on the
+# p-values of the coefficients associated to the potential parents of a node.
+# For instance, consider a graph :math:`G` with 3 vertices :math:`V = \{1, 2, 3\}`. For
+# simplicity let the topological order be trivial, i.e. :math:`\{1, 2, 3\}`. The unique
+# fully connected adjacency matrix compatible with such ordering is the upper
+# triangular matrix `np.triu(np.ones((3, 3)), k=1)` with all ones above the
+# diagonal.
+score = SCORE()  # or DAS() or NoGAM() or CAM()
+score.fit(data, context)
+
+# SCORE estimates a directed acyclic graph (DAG) and the topoological order
+# of the nodes in the graph. SCORE is consistent in the infinite samples
+# limit, meaning that it might return faulty estimates due to the finiteness
+# of the data.
+graph = score.graph_
+order = score.order_
+
+# `score_full_dag.png` visualizes the fully connected DAG representation of
+# the inferred topological ordering.
+# `score_dag.png` visualizes the fully connected DAG after pruning with
+# sparse regression.
+dot_graph = draw(nx.from_numpy_array(full_dag(order), create_using=nx.DiGraph))
+dot_graph.render(outfile="score_full_dag.png", view=True)
+
+dot_graph = draw(graph)
+dot_graph.render(outfile="score_dag.png", view=True)
```

Comment:
You may consider adding titles to each graph so it's apparent what one is looking at.

See: https://www.pywhy.org/pywhy-graphs/dev/auto_examples/visualization/draw_and_compare_graphs_with_same_layout.html#sphx-glr-auto-examples-visualization-draw-and-compare-graphs-with-same-layout-py